src/utils/ui-utils.ts

```diff
@@ -316,4 +316,46 @@ export function convertVWToPixel(value: number) {
 
   const result = (x * value) / 100;
   return result;
+}
+
+/**
+ * Delete cookie helper function
```

Comment:
Explain what happens if the cookie doesn't exist. Also add return type (`void`).